.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           node-version: '20'
       - run: npm ci
       - run: npm test
+      - run: npm run test:e2e
```

Comment:
The e2e test are not ready to run in CI yet, they rely on a running dev server. I will work in that independent of this PR please revert so that the pipeline gets green again.

Reply:
OK

Reply:
removed e2e from CI